src/components/wayback-info.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+/**
+ * @typedef WaybackInfo
+ * @property {Object} fromMetadata
+ * @property {Object} toMetadata
+ * @property {String} pageUrl
+ */
+
+/**
+ * Renders link to see Wayback calendar view of the page
+ * If one or both of the page diffs were sourced from Wayback, this component renders links to the Wayback source url(s)
+ *
+ * @class WaybackInfo
+ * @extends {React.Component}
+ * @param {WaybackInfoProps} props
+ */
+
+export default class WaybackInfo extends React.Component {
+  render () {
+    const waybackCalendarUrl = `https://web.archive.org/web/*/${this.props.pageUrl}`;
+    const pageUrl = <a href={waybackCalendarUrl} target="_blank" rel="noopener noreferrer">See Wayback calendar view</a>;
+    const toUrl = this.props.toMetadata.view_url ? <span> | <a href={this.props.toMetadata.view_url} target="_blank" rel="noopener noreferrer">See Wayback previous page version</a></span> : '';
```

Comment:
This works great, but I’m thinking it might be better to pass the version objects rather than just the metadata, or to only pass the metadata if the version’s `source_type` is `internet_archive`. We could have a future non-Wayback source that we also have `view_url` for.

Reply:
Actually, maybe another solution would be to simply re-word the text here to something like “View previous page version in source system” or “View previous page version in [use mapping to convert `internet_archive` to `Wayback Machine` here]”.

That way, this would work for any source that has a `view_url`, and would show more correct information if the source was something other than Wayback.

(Quick note: in code, we have `internet_archive`, but after that was created, we learned that they’d rather us say “Wayback Machine” since that’s the particular part of the Archive that is concerned with web pages. Hence the difference between code constants and human-readable text on the page.)

Reply:
Sounds good, I will future-proof this component and update the PR. I personally think we should keep some reference to what the source is so the user knows where the link will take them. I will create a map for this.